recipe/meta.yaml

```diff
@@ -31,11 +29,6 @@ outputs:
           - python_abi {{ python_maj_min }} *_pypy{{ python_maj_min.replace(".", "") }}_pp{{ pypy_abi }}
         noarch:
           - {{ pin_subpackage("python", max_pin="x.x", min_pin="x.x") }}
-      script:
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/python
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/python3
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/python{{ python_maj_min }}
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/pypy
```

Comment:
Now done in the packaging step as part of pypy3.6-feedstock calling `python .../package.py`

Reply:
Got it.